src/printer.js

```diff
@@ -811,7 +811,8 @@ function printStatement(path, options, print) {
               concat([",", line]),
               path.map(item => concat([print(item)]), "items")
             ),
-            node.name ? "}" : ""
+            node.name ? "}" : "",
+            ";"
```

Comment:
Can you add a test for this?

Reply:
@czosel we already have ugly output https://github.com/prettier/plugin-php/pull/56/files#diff-07d375fafb7ea50b83dfe5d31b5b2d33L53

Reply:
But i don't see a test for the added semicolon in `usegroup`?

Reply:
Ah sorry, i was misreading the code changes - forget what i said :wink: